api/src/services/report/smsparser.js

```diff
@@ -209,9 +210,22 @@ const fieldParsers = {
   month: (raw) => {
     // keep months integers, not their list value.
     return parseNum(stripInvisibleCharacters(raw));
+  },
+  phone_number: (raw) => {
+    const formattedAndValidatedPhone = phoneNumberParser.normalize(config.getAll(), raw);
+    if (formattedAndValidatedPhone) {
+      return formattedAndValidatedPhone;
```

Comment:
@PrjShrestha , would you mind adding Nepali digit supporting code for this type as well? Here's the sample [PR](https://github.com/medic/cht-core/pull/8450).

Reply:
Done